pkg/noun/manage.c

```diff
@@ -2253,6 +2264,8 @@ u3m_init(size_t len_i)
       exit(1);
     }
 
+    u3t_sstack_init();
```

Comment:
This should go somewhere else, this runs for every loom initialization, including the ivory pill. Maybe this should just go in the initialization of the serf?

Reply:
Please disregard above, fixed the issue